src/DynamoCoreWpf/Utilities/LibraryDragAndDrop.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Dynamo.Search.SearchElements;
+using Dynamo.Wpf.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows;
+
+namespace Dynamo.Wpf.Utilities
+{
+    public class LibraryDragAndDrop
+    {
+        /// <summary>
+        /// Position of mouse, when user clicks on button.
+        /// </summary>
+        public Point StartPosition;
```

Comment:
Should be named `startPosition` and made `private readonly`. Then the only way to initialize this is through a constructor, does that make sense?